Dockerfile.cli

```diff
@@ -0,0 +1,27 @@
+# base Go image version for building the binaries
+FROM golang:1.13.10 AS build
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+ARG version
+
+RUN go build -mod=readonly -o ./bin/mesg-cli -ldflags="-s -w -X 'github.com/cosmos/cosmos-sdk/version.Name=mesg' -X 'github.com/cosmos/cosmos-sdk/version.ServerName=mesg-daemon' -X 'github.com/cosmos/cosmos-sdk/version.ClientName=mesg-cli' -X 'github.com/cosmos/cosmos-sdk/version.Version=$version'" ./cmd/mesg-cli/
+RUN go build -mod=readonly -o ./bin/mesg-daemon -ldflags="-s -w -X 'github.com/cosmos/cosmos-sdk/version.Name=mesg' -X 'github.com/cosmos/cosmos-sdk/version.ServerName=mesg-daemon' -X 'github.com/cosmos/cosmos-sdk/version.ClientName=mesg-cli' -X 'github.com/cosmos/cosmos-sdk/version.Version=$version'" ./cmd/mesg-daemon/
+
+# ubuntu image with binaries for distribution
+FROM ubuntu:18.04
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends ca-certificates=20180409 && \
```

Comment:
do we still need this install? If I remember correctly it was related to the fetch from git that we had but we don't have this anymore.

Reply:
I think it's good to install the `ca certificates` to verify SSL connection.
User could setup a SSL remote connection to another peer or to a RPC endpoint?